### DISCUSSION TITLE
blog/xaringanextra-v0.6.0/

### DISCUSSION THREAD

**Discussion by giscus** [gadenbuie/garrickadenbuie-com#113] [Comments]
# blog/xaringanextra-v0.6.0/

xaringanExtra v0.6.0 is now available on CRAN! Plus some new features.

https://www.garrickadenbuie.com/blog/xaringanextra-v0.6.0/

**Comment by jorgesinval** [comment]
Can the banner include small images? Like the solution provided by Yihui with remark-slide-content...

**Reply by gadenbuie** [reply]
Yup! You can put HTML in the banner, either using `htmltools` or just writing raw HTML. It doesn't convert markdown though, so you'd have to write `'<img src="kitten.png" alt="kitten">'` instead of `[kitten](kitten.png)`. You might also need to fiddle with the `height` attribute of the image, too.

**Reply by jorgesinval** [reply]
Wow! Fantastic!

**Reply by jorgesinval** [reply]
I am using a title banner on each slide, does the function allows me to use it in the title banner?

![image](https://user-images.githubusercontent.com/8278071/173110927-e5ad21cf-fde6-493a-94cf-e917d632c325.png)

**Reply by gadenbuie** [reply]
hmmm that's interesting. I think it might be easier to open an issue over in [xaringanExtra](https://github.com/gadenbuie/xaringanExtra/issues/new) and link to example code or include a small reprex in the issue.

**Reply by gadenbuie** [reply]
FYI for anyone else wondering how to solve such a problem, there's answer here https://github.com/gadenbuie/xaringanExtra/issues/168
